### PR TITLE
[story/ECP-843] Add identity claim verifier admin claim notes claimant vs DQT mismatch information

### PR DESCRIPTION
**Pull request by nomasprime**

### CODE REVIEW

File: app/models/automated_checks/claim_verifiers/identity.rb [hunk 1]
```diff
@@ -42,8 +42,18 @@ def complete_match
...
+      def create_field_note(
+        name:,
+        claimant:,
+        dqt:
+      )
+        body = <<~HTML
+          #{name} not matched:
+          <pre>
+            Claimant: <span class="dark-grey">"</span><span class="red">#{claimant}</span><span class="dark-grey">"</span>
+            DQT:      <span class="dark-grey">"</span><span class="green">#{dqt}</span><span class="dark-grey">"</span>
```
Comment (WorkSutton) [inline app/models/automated_checks/claim_verifiers/identity.rb:54]: This feels presentational, not sure why in model.

File: spec/models/automated_checks/claim_verifiers/identity_spec.rb [hunk 2]
```diff
@@ -87,767 +88,847 @@ module ClaimVerifiers
...
 
-              describe "#created_by" do
-                subject(:created_by) { note.created_by }
+                    it do
+                      is_expected.to eq(
+                        <<~HTML
+                          National Insurance number not matched:
+                          <pre>
+                            Claimant: <span class="dark-grey">"</span><span class="red">QQ100000C</span><span class="dark-grey">"</span>
+                            DQT:      <span class="dark-grey">"</span><span class="green">QQ100000B</span><span class="dark-grey">"</span>
```
Comment (WorkSutton) [inline spec/models/automated_checks/claim_verifiers/identity_spec.rb:203]: Why not in view or presentation object? Not a big fan of seeing presentation in the model layer

**Review by WorkSutton — APPROVED** [review]
Approving due to time constraints, would like in future to consider why we need presentation in the model

File: app/models/automated_checks/claim_verifiers/identity.rb (hunk 1 above)
Comment (nomasprime) [reply app/models/automated_checks/claim_verifiers/identity.rb:54]: Speed. Refactor later 👍

**Pull request merged**